src/docs/tutorial-branding/working-with-translations/index.md

```diff
@@ -1,42 +1,66 @@
 ---
 title: Working with translations
 slug: working-with-translations
-updated: 2020-02-28
+updated: 2022-05-13
 category: tutorial-branding
 ingress: Change the UI copy-texts and implement new translations.
 published: true
 ---
 
 ## Where to find copy-texts
 
-One of the biggest mandatory tasks for customization project is to
+One of the biggest mandatory tasks for a customization project is to
 change the UI texts. FTW-daily is a rental marketplace for saunas - so,
 there are quite many components and pages that need to change the
-copy-texts. We refer to copy-text as translations, since there could be
-different translation files used instead of the default one: _en.json_.
+copy-texts.
 
-Most of the translations can be found from this _en.json_ file:
+The easiest way to modify the copy texts is through Flex Console, in
+Build > Content.
+
+![Modify marketplace texts](./translation_start.png)
+
+_**Note:** Hosted assets are available in Flex and the FTW templates
+starting from version XX.XX. If you have an earlier version and want to
```

Comment:
Related to this, I think the next version are:
- FTW-daily v8.5
- FTW-hourly v10.5
- FTW-product: v9.2

aka minor version update